[2025-03-30T17:50:47.723+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-03-30T17:50:47.739+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:50:39.564532+00:00 [queued]>
[2025-03-30T17:50:47.743+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:50:39.564532+00:00 [queued]>
[2025-03-30T17:50:47.744+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-03-30T17:50:47.758+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): create_hive_database> on 2025-03-30 17:50:39.564532+00:00
[2025-03-30T17:50:47.763+0000] {standard_task_runner.py:64} INFO - Started process 205 to run task
[2025-03-30T17:50:47.766+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'dag_deliverable', 'create_hive_database', 'manual__2025-03-30T17:50:39.564532+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/dag_deliverable.py', '--cfg-path', '/tmp/tmpfxmu2ciq']
[2025-03-30T17:50:47.767+0000] {standard_task_runner.py:91} INFO - Job 152: Subtask create_hive_database
[2025-03-30T17:50:47.793+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_deliverable.create_hive_database manual__2025-03-30T17:50:39.564532+00:00 [running]> on host airflow
[2025-03-30T17:50:47.871+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_deliverable' AIRFLOW_CTX_TASK_ID='create_hive_database' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T17:50:39.564532+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-30T17:50:39.564532+00:00'
[2025-03-30T17:50:47.872+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-03-30T17:50:47.889+0000] {base.py:84} INFO - Using connection ID 'hive_default' for task execution.
[2025-03-30T17:50:47.921+0000] {hive.py:475} INFO - USE `default`
[2025-03-30T17:50:47.941+0000] {hive.py:475} INFO - CREATE DATABASE IF NOT EXISTS weather
[2025-03-30T17:50:47.949+0000] {dag_deliverable.py:100} INFO - âœ… Base de datos 'weather' creada correctamente en Hive
[2025-03-30T17:50:47.949+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-03-30T17:50:47.950+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-03-30T17:50:47.953+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_deliverable, task_id=create_hive_database, run_id=manual__2025-03-30T17:50:39.564532+00:00, execution_date=20250330T175039, start_date=20250330T175047, end_date=20250330T175047
[2025-03-30T17:50:47.978+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-03-30T17:50:47.989+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-30T17:50:47.995+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
